src/OpenFeature.Contrib.Providers.Flagd/Resolver/InProcess/JsonEvaluator.cs

```diff
@@ -234,31 +247,31 @@ private ResolutionDetails<T> ResolveValue<T>(string flagKey, T defaultValue,
                     flagConfiguration.Targeting.ToString() != "{}")
                 {
                     reason = Reason.TargetingMatch;
-                    var flagdProperties = new Dictionary<string, Value>();
-                    flagdProperties.Add(FlagdProperties.FlagKeyKey, new Value(flagKey));
-                    flagdProperties.Add(FlagdProperties.TimestampKey,
-                        new Value(DateTimeOffset.UtcNow.ToUnixTimeSeconds()));
+                    var flagdProperties = new Dictionary<string, Value>
+                    {
+                      { FlagdProperties.FlagKeyKey, new Value(flagKey) },
+                      { FlagdProperties.TimestampKey, new Value(DateTimeOffset.UtcNow.ToUnixTimeSeconds()) }
+                    };
 
                     if (context == null)
                     {
                         context = EvaluationContext.Builder().Build();
                     }
 
-                    var targetingContext = context.AsDictionary().Add(
-                        FlagdProperties.FlagdPropertiesKey,
-                        new Value(new Structure(flagdProperties))
-                    );
 
+                    var contextDictionary = context.AsDictionary();
+                    contextDictionary = contextDictionary.Add(FlagdProperties.FlagdPropertiesKey, new Value(new Structure(flagdProperties)));
+                    // TODO: all missing comments
```

Comment:
Another TODO here. Is it still relevant?

Reply:
Already resovled this one, forgot to delete.